recipe/meta.yaml

```
@@ -421,28 +423,28 @@ outputs:
         {% endfor %}
         - |
           source ${PREFIX}/etc/conda/activate.d/{{ PKG_NAME }}_activate.sh
-          test -n "${OMNISCI_ROOT_PATH+x}"
+          test -n "${HEAVYAI_ROOT_PATH+x}"
           ldd ${PREFIX}/lib/libDBEngine.so
 
     about:
-      home: https://www.omnisci.com/
+      home: https://heavy.ai
       license: Apache-2.0
       license_family: APACHE
       license_file: LICENSE.md
-      summary: The OmniSci database
+      summary: The HeavyDB database
 
       description: |
-        OmniSciDB is an in-memory, column store, SQL relational database
+        HeavyDB is an in-memory, column store, SQL relational database
         that was designed from the ground up to run on GPUs.
-      doc_url: https://www.omnisci.com/docs/latest/
-      dev_url: https://github.com/omnisci/omniscidb
+      doc_url: https://docs.heavy.ai/
+      dev_url: https://github.com/heavyai/heavydb
 
-  - name: pyomniscidbe
+  - name: pyheavydbe
     version: {{ version }}
-    script: build-omniscidb.sh
+    script: build-heavydb.sh
     build:
       string: {{ py_build_string_ext }}
-      skip: true  # [cuda_compiler_version not in (undefined, "None", "11.0") or not linux64]
+      skip: true
```

A meant to keep the original skip as we'll need it later:
```suggestion
      skip: true  # [cuda_compiler_version not in (undefined, "None", "11.0") or not linux64]
      skip: true  # temporarily skipping pyheavydbe build
```
(one can use multiple skip lines in one build section).